### PR TITLE
[MIG] stock_putaway_method

### PR DESCRIPTION
**Pull request by max3903**

### CODE REVIEW

File: stock_putaway_method/views/product_strategy_views.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+    <record id="view_putaway" model="ir.ui.view">
+        <field name="name">product.putaway.form.method</field>
+        <field name="model">product.putaway</field>
+        <field name="inherit_id" ref="stock.view_putaway"/>
+        <field name="arch" type="xml">
+            <field name="name" position="after">
+                <field name="method"/>
+            </field>
+            <xpath expr="//separator" position="attributes">
```
Comment (nikul-serpentcs) [inline stock_putaway_method/views/product_strategy_views.xml]: @max3903 `//separator` was removed in v12, So no need to added `attributes`

**Review by bodedra — APPROVED** [review]
LGTM :+1:

**Comment by rousseldenis** [comment]
@max3903 Could you squash commits ?

**Review by rousseldenis — APPROVED** [review]
Code review

**Comment by bodedra** [comment]
> @max3903 Could you squash commits ?

@rousseldenis  It's done. Thank you.

**Review by osi-scampbell — APPROVED** [review]

**Comment by josepmy** [comment]
It's ready stock_putaway_method?

**Pull request merged**